Rubberduck.Refactorings/Common/IModuleRewriterExtensions.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Antlr4.Runtime;
+using Rubberduck.Parsing;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Rewriter;
+using Rubberduck.Parsing.Symbols;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rubberduck.Refactorings.Common
+{
+    public static class IModuleRewriterExtensions
```

Comment:
Since RD 'owns' `ModuleRewriter`, adding the methods to `IModuleRewriter` is an available choice and certainly avoids the drawbacks you mention.  Even though I used an extension class, I'm not advocating that extension methods are the _best_ choice here.  For a PR submission, I am very reluctant to modify such a ubiquitous interface as `IModuleRewriter` without specific direction...So, extension methods provide a less impactful way to initially add/propose new behavior.  I'm good with making the extension methods part of the `IModuleRewriter` if that is preferred.